frontend/packages/shared/src/components/GiteaHeader/ThreeDotsMenu/CloneModal/CloneModal.test.tsx

```diff
@@ -26,19 +26,22 @@ const render = (props: Partial<ICloneModalProps> = {}) => {
 
 describe('cloneModal', () => {
   it('should show copy link if copy feature is supported', () => {
+    const consoleErrorSpy = jest.spyOn(console, 'error').mockImplementation(() => {});
```

Comment:
Dette bør vi ikke gjøre med mindre testen faktisk skal sjekke om det kommer en feil. Hvis det kommer feilmeldinger her, bør vi heller prøve å løse feilen. Kommer det på grunn av ikonendringene du har gjort?

Reply:
har fjernet den, det var litt rart med den komponenten, ikke helt sikkert at det er pga endringer. men den tester grønn nå.

Reply:
Da må du sammenligne med master for å se om det var der fra før. Hvis det var der, kan du ignorere det nå siden det ikke inngår i oppgaven, men vi bør så klart fikse det på sikt.

Reply:
nei den consoleErrorSpy  var ikke , jeg lagt den før , men har fjernet den og blir ikke noe endringer i den komponent , den er er samme som master og kjørte grønn